.github/workflows/parachain.yml

```diff
@@ -73,29 +73,19 @@ jobs:
         run: rustup show
       # Increase stack limit for beacon light client tests
       - run: sudo prlimit --pid $$ --stack=32768
-      # Run tests for everything except the beacon light client
-      - name: Tests for everything except the beacon light client
+      # Run tests with feature benchmark
+      - name: Tests with benchmark 
         run: >
           cargo test
           --manifest-path parachain/Cargo.toml
           --workspace
           --features runtime-benchmarks
-          --exclude snowbridge-query-events
-          --exclude snowbridge-ethereum-beacon-client
-      # Run tests for beacon light client, excluding benchmark tests
-      - name: Tests for beacon light client excluding benchmark tests
+      # Run tests for beacon light client, excluding benchmark tests(specific for minimal spec)
+      - name: Tests for beacon light client for minimal spec
```

Comment:
Some cleanup in CI Remove Benchmark tests for beacon light client it's not necessary.

Reply:
I think it is good to run benchmark tests for pallets in CI. It lets us know if we broke benchmarks.

Reply:
Yes, we run benchmarks in CI for all the packages including beacon light client.  Just remove the previous task specific for beacon light client which is duplicated.

Reply:
I don't see any duplication. There are two sets of test we need to run for the beacon light client:

* Normal tests
* Tests for benchmarks

Those tests need to be run separately as they require different feature flags (beacon-spec-minimal enabled for disabled)

Reply:
Sorry if any misleading, not a duplication just merge the previous two tasks into one

- https://github.com/Snowfork/snowbridge/blob/6b08d309db4a7f5a7ec20ae490173644b6ea47ff/.github/workflows/parachain.yml#L77C15-L77C66

- https://github.com/Snowfork/snowbridge/blob/6b08d309db4a7f5a7ec20ae490173644b6ea47ff/.github/workflows/parachain.yml#L92

Reply:
If you can show me CI logs that prove that normal tests and benchmark tests for the beacon client are running I will be happy. I took a look and I wasn't convinced.

The various feature flag magic shown around the code below does complicate the test modes:

https://github.com/Snowfork/snowbridge/blob/1ffd01ec450ba06c5d7562e431b03b50c3c55c2c/parachain/pallets/ethereum-beacon-client/src/lib.rs#L18

Reply:
https://github.com/Snowfork/snowbridge/actions/runs/6169002603/job/16742527677#step:6:737

```
running 2 tests
test mock::mainnet::__construct_runtime_integrity_test::runtime_integrity_tests ... ok
test benchmarking::benchmarks::benchmark_tests::test_benchmarks ... ok

test result: ok. 2 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in 7.77s
```